uml_diagram_link.txt

```diff
@@ -0,0 +1 @@
+https://drive.google.com/file/d/1ftGqfRhbvMWFVt5fAMi9WsDR4Nxqupbj/view?usp=sharing
```

Comment:
DRFactory class is duplicated. Dependencies from DRFactory should be <<creates>> not <<use>>.